lib/kino_flame/runner_cell.ex

```diff
@@ -126,6 +222,33 @@ defmodule KinoFLAME.RunnerCell do
 
     env = {:%{}, [], envs}
 
+    backend_ast =
+      quote do
+        {FLAME.FlyBackend,
+         [
+           unquote_splicing(specs_opts),
+           env: unquote(env)
+         ]}
+      end
+
+    to_quoted_pool(attrs, backend_ast)
+  end
+
+  defp to_quoted(%{"backend" => "k8s"} = attrs) do
+    multiline_runner_pod_template =
+      {:sigil_y, [delimiter: ~S["""]], [{:<<>>, [], [attrs["runner_pod_template"]]}, []]}
```

Comment:
Sorry, I am slightly confused about where we landed.

The smart cell definition on Livebook (https://github.com/livebook-dev/livebook/pull/2762) no longer adds `yaml_elixir`. In the test below, we also use the `~s` sigil. So do we need the dependency or not? I thought that flame_k8s_backend would send the template as is to k8s?

If we don't need `YamlElixir`, can we just go ahead and yank all references to it? :) Thank you!

Reply:
Happy to clarify: 

Yes, the two pull requests are currently off sync. This is because I want to come to an agreement on this topic before pushing all the changes everywhere.

> I thought that flame_k8s_backend would send the template as is to k8s? If we don't need YamlElixir, can we just go ahead and yank all references to it? :) Thank you!

The Kubernetes API Server expects JSON. While JSON is valid YAML, the opposite is not the case. So we need to parse the YAML binary into a map somewhere. I'd rather have this being done by the user (i.e. in the livebook runtime or the application) than adding `yaml_elixir` as a hard dependency to either the `flame_k8s_backend` or `kino_flame`.

Reply:
Ok, I think the current approach is the most sane.

@josevalim see https://github.com/livebook-dev/kino_flame/pull/3#discussion_r1734637009, let me know if you have any other option in mind :)